### ISSUE TITLE
bug? ~ doc.deno.land is empty with normal local `deno docs` (for 'os_paths')

### ISSUE THREAD

**Issue by rivy** [denoland/doc_website#161]
For ['os_paths'](https://deno.land/x/os_paths@v6.6.0), doc.deno.land shows an [empty page](https://doc.deno.land/https/deno.land/x/os_paths@v6.6.0/src/mod.deno.ts) for the [*src/mod.deno.ts* file](https://deno.land/x/os_paths@v6.6.0/src/mod.deno.ts) where locally `deno doc src/mod.deno.ts` shows:

```text
Defined in file:///C:/Users/Roy/OneDrive/Projects/js/os-paths/repo.js.os-paths/src/mod.deno.ts:15:0

var default

Defined in file:///C:/Users/Roy/OneDrive/Projects/js/os-paths/repo.js.os-paths/dist/types/mod.d.ts:2:0

interface OSPaths
  `OSPaths` (API) Determine common OS/platform paths (home, temp, ...)

  new()
    Create an `OSPaths` object (`new` is optional).
  home(): string | undefined
    Returns the path string of the user's home directory (or `undefined` if the user's home directory is not resolvable).
  temp(): string
    Returns the path string of the system's default directory for temporary files.
```

Am I hitting some sort of limitation? Or is this a bug?
Thanks for looking at this...

**Comment by Liamolucko** [comment]
It's because `doc.deno.land` is still using deno 1.6.2 - if I downgrade to 1.6.2 the CLI also shows nothing.

It was attempted to be updated in #158 but wasn't working.

**Comment by rivy** [comment]
Thanks for the look.
I'll await the planned fix (#162).
